# HG changeset patch
# User Rik <rik@octave.org>
# Date 1531256464 25200
#      Tue Jul 10 14:01:04 2018 -0700
# Branch stable
# Node ID 0e09d5fd7dab12075bc5b3fcb3df9da2dc1e3c4a
# Parent  64bcc3ab2f6a2f9ffc21d8bd2e7f6d574fca93d9
doc: Clarify description of Example 4 for size() (bug #54283).

* data.cc (Fsize): Clarify that Example 4 applies when the number
of output arguments is less than the number of dimensions.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -2651,17 +2651,17 @@ Example 3: number of output arguments ==
 @example
 @group
 [nr, nc] = size ([1, 2; 3, 4; 5, 6])
     @result{} nr = 3
     @result{} nc = 2
 @end group
 @end example
 
-Example 4: number of output arguments != number of dimensions
+Example 4: number of output arguments < number of dimensions
 
 @example
 @group
 [nr, remainder] = size (ones (2, 3, 4, 5))
     @result{} nr = 2
     @result{} remainder = 60
 @end group
 @end example
